android/src/main/java/com/heapanalytics/reactnative/HeapInitializationProvider.java

```diff
@@ -0,0 +1,50 @@
+package com.heapanalytics.reactnative;
+
+import android.content.ContentProvider;
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.net.Uri;
+
+public class HeapInitializationProvider extends ContentProvider {
+    @Override
+    public boolean onCreate() {
+        String appId;
+
+        if (BuildConfig.DEBUG) {
+            appId = BuildConfig.HEAP_APP_ID_DEV;
+        } else {
+            appId = BuildConfig.HEAP_APP_ID_PROD;
+        }
+
+        if (!appId.isEmpty()) {
+            RNHeap.init(getContext(), appId);
```

Comment:
Ah, I see... so this init pathway will ONLY get hit if we HAVE a valid BuildConfig.  Never mind about my comment about unifying those two calls then!